front/src/config/i18n/fr.json

```diff
@@ -1472,6 +1473,14 @@
     "invalidIcon": "L'icône est requise",
     "sceneAlreadyExist": "Une scène avec le même nom existe déjà."
   },
+  "duplicateScene": {
+    "cardTitle": "Copier une scène",
```

Comment:
The text could be dynamic and indicate which scene is being duplicated, because here we lose the information of which scene is being duplicated